src/components/Pagination/Pagination.tsx

```diff
@@ -24,9 +24,15 @@ export interface PaginationDescriptor {
   nextURL?: string;
   /** The URL of the previous page */
   previousURL?: string;
-  /** Whether there is a next page to show */
+  /**
+   * Whether there is a next page to show
+   * @default true
+   */
   hasNext?: boolean;
-  /** Whether there is a previous page to show */
+  /**
+   * Whether there is a previous page to show
+   * @default true
```

Comment:
Why does this default to true?

Reply:
I was to quick on adding this, percy should be fixed.